src/main/kotlin/org/rust/ide/formatter/RustSpacing.kt

```diff
@@ -0,0 +1,118 @@
+package org.rust.ide.formatter
+
+import com.intellij.formatting.SpacingBuilder
+import com.intellij.psi.codeStyle.CommonCodeStyleSettings
+import com.intellij.psi.tree.IElementType
+import com.intellij.psi.tree.TokenSet
+import org.rust.lang.core.psi.RustCompositeElementTypes.*
+import org.rust.lang.core.psi.RustKeywordTokenType
+import org.rust.lang.core.psi.RustTokenElementTypes.*
+import com.intellij.psi.tree.TokenSet.create as ts
```

Comment:
It's pretty much just an alias for `TokenSet.create`. I'd stick with simple renaming in import as this is Kotlin and it's possible :smile: